### ISSUE TITLE
OrbitControl behaves strangely after first drag

### ISSUE THREAD

**Issue by mathieu1** [maartenbreddels/ipyvolume#89]
Using `fig.camera_control = 'orbit'`, the Orbital Control works as expected at first glance. However, after dragging the object and "tilting the horizontal plane" to a "non-horizontal position", the orbit controls do not behave as expected anymore, and do not keep the Y axis upwards (contrary to the standard behaviour of threejs' OrbitControl)

Hopefully what I mean is clarified with this:
![45zkr-ev7z3](https://user-images.githubusercontent.com/7703352/34018288-7cb0a714-e129-11e7-9965-e5dd869e68c0.gif)

**Comment by maartenbreddels** [comment]
Thanks, and this is only using *that* camera control right? Not switching between them?

**Comment by mathieu1** [comment]
Indeed, nothing else happens in that clip besides two consecutive mouse drags. The code has nothing special AFAICT (nevermind the x,y,z rearrangement):
```python
fig = p3.figure()
fig.camera_control = 'orbit' 
p3.plot_trisurf(y, z, x, triangles=faces, color=colors)
p3.show()
```

**Comment by Carreau** [comment]
Same for me, regardless of whether `orbit` is set from the python side, or by clicking the "up arrow" button. (BTW, I would suggest changing it for "globe") that would be IMHO more representative.

**Comment by mgeier** [comment]
@mathieu1 and @Carreau Does it work better with #273?
